hnn_core/cell.py

```diff
@@ -148,6 +148,51 @@ def gid(self, gid):
             raise RuntimeError('Global ID for this cell already assigned!')
 
 
+class Section:
+    """Section class.
+
+    Parameters
+    ----------
+    L : float
+        length of a section in microns
+    diam : float
+        diameter of a section in microns
+    cm : float
+        membrane capacitance in micro-Farads
+    Ra : float
+        axial resistivity in ohm-cm
+    """
+    def __init__(self, L, diam, Ra, cm, sec_pts=None):
+
+        self._L = L
+        self.diam = diam
+        self.Ra = Ra
+        self.cm = cm
+        if sec_pts is None:
+            sec_pts = list()
+        self.sec_pts = sec_pts
```

Comment:
We may want to save this for another PR , but what do we think of renaming `sec_pts` to `end_pts` or `end_pts_3d` now that this is an attribute of `Section`?

Reply:
Isn't `sec_pts` actually a list of two 3D coordinates: (start, end)

Reply:
Yes it is, though I'm not sure I follow what you're trying to point out

Reply:
semantics, `end_pts` is fine